app/assets/stylesheets/valiables.scss

```diff
@@ -0,0 +1,7 @@
+$text-color-banner: #FEFEFE;
+$back-ground-color-primary: #D7D5D2;
+$title-color: #5E4633;
+$list-room-item-background-hover: #ACB9C8;
+$back-ground-color-footer: #333333;
+$text-color-footer: #AAAAAA;
+$text-color-footer-social: #777777;
```

Comment:
với những mãu màu mà ký tự giống nhau hoàn toàn thì viết 3 ký tự thôi, ví dụ `#777777` => `#777`